warehouse/query-core/src/main/java/datawave/query/cardinality/CardinalityRecord.java

```diff
@@ -221,15 +221,12 @@ public void writeToDisk(final File file) {
     public static CardinalityRecord readFromDisk(File file) {
         
         CardinalityRecord cardinalityRecord = null;
-        ObjectInputStream ois = null;
-        try {
-            FileInputStream fis = new FileInputStream(file);
-            ois = new ObjectInputStream(fis);
+
+        try (FileInputStream fis = new FileInputStream(file)) {
+            ObjectInputStream ois = new ObjectInputStream(fis);
```

Comment:
What happened to closing the ois?  If the ois does any buffering, then we may get in trouble here.

Reply:
fixe din next commit